src/checks.py

```diff
@@ -1,10 +1,11 @@
 import filecmp
+import functools
 import logging
 import os
 from typing import Callable, List, Optional, Union
 
 import apache_2_license
-from helpers import format_check_name, sh, step
+from helpers import sh, step
```

Comment:
According to PEP-8 imports should be one by line. `flake8` should complain about this...

Reply:
Negative. `import $MODULE` statements should be one by line; `from $MODULE import $a, $b` is fine (refer to https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
Oh you're right. Feel free to ignore this then